client/my-sites/sidebar-navigation/style.scss

```diff
@@ -26,8 +26,6 @@
 	}
 
 	.site-icon {
-		background: lighten( $gray, 20% );
```

Comment:
Oh, and since it's brought up every time I share a screenshot, my site icon is circular with a transparent background, this is not the result of any `border-radius: 50%;` or otherwise to style as circular.